### ISSUE TITLE
Calculation from 'raw' Dihedrals giving artifact structures.

### ISSUE THREAD

**Issue by n4ndoz** [aqlaboratory/pnerf#1]
Hi, as with the Protein Net dataset, again I must thank you for the wonderful work, Dr. AlQuraishi.

My Master's project is centered around differentiable protein structure prediction, and I am trying to insert the pNeRF methods inside my model. It is written on Keras, so I had to tweak a few parameters. 

But while testing the (original) pNeRF code, using already calculated dihedral angles, I was unable to rebuild an structure (also derived from ProteinNet's 7 test set). The resulting structure, when written into a pdb file and open using a visualization software (i.e. PyMol) is nothing but an artifact. 

I'm performing the following routine:

`# shape [BATCH, SEQLEN, N_DIHEDRALS]`
`dihedrals = np.load("dih_0001.npy")`
`# shape [SEQLEN, BATCH, DIHEDRALS]`
`dihedrals = dihdrals.transpose((1,0,2))`
`points = dihedral_to_points(dihedrals)`
`coords = point_to_coordinate(points)`
`write_pdb(coords) #this function is written by me`

I am using Tensorflow 2.2 rc2, running on eager execution and have not tested on different versions of TF for now. But your methods were supposed to work out of the box on such versions (even with scope settings). I know pNeRF isn't supposed to be executed this way, but is there anything wrong in the way I run, or regarding the shapes? All the angles are already in Radians. 

I cannot say that I get the same error/misbehavior when implementing pNeRF as a keras layer, but my model will get sky high losses, so I guess it is not assembling the structures correctly. Do you have any clues of what to do?
